src/ui/navigation.R

```diff
@@ -21,6 +21,9 @@ panelWithTitle <- function(title, content) {
 
 # Server side
 enableNavLinks <- function(input, output, spectrumFilesState, modelRunState) {
+    enableTabWhen("Upload survey data", function() { spectrumFilesState$anyDataSets() })
+    enableTabWhen("Upload programmatic data", function() { spectrumFilesState$anyDataSets() })
+    enableTabWhen("Review input data", function() { spectrumFilesState$anyDataSets() })
```

Comment:
Also changed this to avoid errors when trying to view survey/program data in the absence of a country